Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -1594,4 +1593,28 @@ NOTE: Restart is required for the setting to take effect.</value>
     <value>Declaration type of target '{0}' is '{1}' instead of one of the expected '{2}'.</value>
     <comment>{0}: name of target; {1}: actual declaration type; {2}: expected declaration types</comment>
   </data>
+  <data name="RegexAssistant_EncloseWhitespaceFlag" xml:space="preserve">
+    <value>Enclose whitespace in identifier</value>
```

Comment:
Nitpick: I think that should be changed to `Spell out whitespace in the description`

Reply:
Not a nit at all. Pure oversight of mine.